katas/langtonant/maria/langtonant.js

```diff
@@ -0,0 +1,75 @@
+var gridSize = process.argv[2];
+var steps = process.argv[3];
+
+var gridRow = new Array();
+for (let x = 0; x < gridSize; x++) {
+    gridRow[x] = new Array();
+    for (let y = 0; y < gridSize; y++) {
+        gridRow[x][y] = 0;
+    }
+}
+
+var direction = 'up';
+var positionX = Math.trunc(gridSize / 2);
+var positionY = Math.trunc(gridSize / 2);
+var color = 0;
+while (steps > 0) {
+    if (positionX >= 0 && positionY >= 0) {
+        if (direction == 'up') {
+            if (color == 0) {
```

Comment:
In order to add more clarity to the code, it would be good to declare a constant for '0' value, so the code reads something like
```
const WHITE = 0;

if (color == WHITE)
```

Reply:
i consider this point and fixed it.